sim/midas/src/main/scala/midas/passes/fame/FAMETransform.scala

```diff
@@ -147,35 +147,82 @@ object FAMEModuleTransformer {
      */
     val nReset = DoPrim(PrimOps.Not, Seq(WRef(hostReset)), Seq.empty, BoolType)
 
+    /**
+      * Bundles together four critical elements for managing multiclock processing.
+      *
+      * @param Port The reference to the original target port
+      *
+      * @param outputChannelEnable A boolean that, when asserted alongside
+      * targetCycleFinishing, serves to reset output channels in the associated
+      * clock domain so that new output tokens may be enqueued.
+      *
+      * @param inputChannelEnable Ditto, above. This is asserted one pipeline
+      * stage earlier, ensuring that all inputs are dequeued synchronously with
+      * the launching of the target clock.
+      *
+      * @param clockBuffer Holds a reference to the gated target-clock (+ associated statements)
+      */
+    case class TargetClockMetadata(
+      targetSourcePort: Port,
+      outputChannelEnable: Expression,
+      inputChannelEnable: Expression,
+      clockBuffer: SignalInfo)
+
     // Multi-clock management step 4: Generate clock buffers for all target clocks
-    val targetClockBufs: Seq[SignalInfo] = clockChannel.ports.map { en =>
-      val enableReg = hostFlagReg(s"${en.name}_enabled", resetVal = UIntLiteral(1))
+    val clockMetadata: Seq[TargetClockMetadata] = clockChannel.ports.map { en =>
+      val enableReg = hostFlagReg(s"${en.name}_enabled")//, resetVal = UIntLiteral(1))
```

Comment:
```suggestion
      val enableReg = hostFlagReg(s"${en.name}_enabled")
```

Leaving it commented out makes it look like you're aiming for "don't care," when I believe you are actually relying on the change to the default of zero?